scripts/step03_explore_examples/02_advanced_rag/src/ingest.py

```diff
@@ -76,11 +76,43 @@ def ingest_sap_docs():
     db.add_documents(chunks)
     log.success("Added SAP btp docs successfully!")
 
+def load_terraform_docs_from_github():
+    # Load the documents from a GitHub repository
+    loader = GitLoader(
+        clone_url="https://github.com/SAP/terraform-provider-btp",
+        repo_path="./gen/docs/",
+        file_filter=lambda file_path: file_path.startswith("./gen/docs/docs")
+        and file_path.endswith(".md"),
+        branch="main",
+    )
+    text_documents = loader.load()
+    log.info("Getting the Terraform documents from the GitHub repository ...")
+
+    # Split the documents into chunks
+    chunks = split_docs_into_chunks(documents=text_documents)
+    return chunks    
+
+def ingest_terraform_docs():
```

Comment:
This Function repeats the functionality, please consider moving it to [Utils](https://github.com/SAP-samples/btp-genai-starter-kit/tree/main/scripts/step03_explore_examples/utils/utils): https://github.com/SAP-samples/btp-genai-starter-kit/blob/50e3d13e0a9e9bdf2a7eff4221149986a24cab25/scripts/step03_explore_examples/01_naive_rag/library/ingest.py#L36